### ISSUE TITLE
Уместить панели инструментов

### ISSUE THREAD

**Issue by drnextgis** [nextgis/nextgismanager#146]
Можно ли сделать так, чтобы инструмент `Уместить панели инструментов` вызывался автоматически при загрузке менеджера.

**Comment by BishopGIS** [comment]
Это такой способ обойти "баг" общей библиотеки. Можно попробовать. Хотя в свете перехода на qt - не вижу смысла.

**Comment by drnextgis** [comment]
А правильно я понимаю, что если один раз нажать, то при следующем запуске панели будут уже умещенными и каждый раз это не нужно повторять? Если так, то, действительно, нет смысла, если нет - то есть.

**Comment by BishopGIS** [comment]
Да, местоположение панелей и приклепляющихся окон, диалогов - запоминается при выходе из программы.
